src/hooks/useCheckRevision.ts

```diff
@@ -11,18 +13,17 @@ const useCheckRevision = () => {
     const currentIndex = checked.indexOf(revisionID);
```

Comment:
optional: It would be clearer to have a boolean if you dont use the index itself. You can use `const isAlreadyChecked = checked.includes(revisionID)` for example.